niworkflows/interfaces/utils.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Utilities
+
+"""
+
+import nibabel as nb
+from nipype.utils.filemanip import fname_presuffix
+from nipype.interfaces.base import File, OutputMultiPath, BaseInterfaceInputSpec, TraitedSpec
+from .base import SimpleInterface
+
+class CopyHeaderInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='the file we get the data from')
+    hdr_file = File(exists=True, mandatory=True, desc='the file we get the header from')
+
+class CopyHeaderOutputSpec(TraitedSpec):
+    out_file = OutputMultiPath(File(exists=True, desc='written file path'))
+
+class CopyHeader(SimpleInterface):
+    input_spec = CopyHeaderInputSpec
+    output_spec = CopyHeaderOutputSpec
+
+    def _run_interface(self, runtime):
+        in_img = nb.load(self.inputs.hdr_file)
+        out_img = nb.load(self.inputs.in_file, mmap=False)
```

Comment:
This is fine, but since we're not overwriting the file, `mmap=False` isn't necessary. (In practice, its value doesn't matter since we're always using `.nii.gz`, which precludes `mmap`ing.)